etl/processor/processor_test.go

```diff
@@ -0,0 +1,51 @@
+package processor
+
+import (
+	"context"
+	"testing"
+
+	"github.com/insolar/block-explorer/etl/models"
+
+	"github.com/insolar/block-explorer/etl/interfaces/mock"
+	"github.com/insolar/block-explorer/etl/types"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNewProcessor(t *testing.T) {
+
+	ctx := context.Background()
+	JDC := make(chan types.JetDrop)
+	trm := mock.NewTransformerMock(t)
+	trm.GetJetDropsChannelMock.Set(func() (ch1 <-chan types.JetDrop) {
+		return JDC
+	})
+
+	sm := mock.NewStorageMock(t)
+	sm.SaveJetDropDataMock.Set(func(jetDrop models.JetDrop, records []models.Record) (err error) {
```

Comment:
To check that process was done correctly, check that this mock was called as many times as you expected